VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +170,50 @@ public override void Validate()
             return result;
         }
 
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            var languageCodes = ConfigurationHelper.GetAppSettingsValue("VirtoCommerce:Localization:ServerLocalizationCodes");
+
+            foreach (var languageCode in languageCodes.Split(','))
+            {
+                if (!string.IsNullOrEmpty(languageCode))
+                {
+                    var separator = Path.DirectorySeparatorChar;
+                    var localizationDirectoryPath = Path.Combine(sourceDirectoryPath, $"{languageCode}{separator}");
+                    var localizationTargetDirectoryPath = Path.Combine(targetDirectoryPath, $"{languageCode}{separator}");
+
+                    CopyAssemblyFiles(localizationTargetDirectoryPath, localizationDirectoryPath);
+                }
+            }
+        }
+
         private static void CopyAssemblies(string sourceParentPath, string targetDirectoryPath)
         {
             if (sourceParentPath != null)
             {
-                var sourceDirectoryPath = Path.Combine(sourceParentPath, "bin\\");
+                var separator = Path.DirectorySeparatorChar;
+                var sourceDirectoryPath = Path.Combine(sourceParentPath, $"bin{separator}");
 
-                if (Directory.Exists(sourceDirectoryPath))
-                {
-                    var sourceDirectoryUri = new Uri(sourceDirectoryPath);
+                CopyLocalizationAssemblies(sourceDirectoryPath, targetDirectoryPath);
+
+                CopyAssemblyFiles(targetDirectoryPath, sourceDirectoryPath);
+            }
+        }
 
-                    foreach (var sourceFilePath in Directory.EnumerateFiles(sourceDirectoryPath))
+        private static void CopyAssemblyFiles(string targetDirectoryPath, string sourceDirectoryPath)
```

Comment:
Lets order method parameters in the following way (as in other places):
source, target

Reply:
fixed